### ISSUE TITLE
neutrino-preset-jest says "no tests found"

### ISSUE THREAD

**Issue by jefo** [mozilla-neutrino/neutrino-dev#518]
I have two test files, but preset says "No tests found". In same time karma preset works well if i replace jest in `--use`.
Windows 10, Node 8.9.0
```
PS C:\Users\j4\Projects\rxsupp.admin> yarn test
yarn run v1.2.1
$ neutrino test --use neutrino-preset-react neutrino-preset-jest
No tests found
In C:\Users\j4\Projects\rxsupp.admin
  21 files checked.
  testMatch:  - 21 matches
  testPathIgnorePatterns: \\node_modules\\ - 21 matches
  testRegex: test\.*(_test|_spec|\.test|\.spec)\.jsx?$ - 0 matches
Pattern:  - 0 matches
Done in 6.76s.
PS C:\Users\j4\Projects\rxsupp.admin> ls test


    Каталог: C:\Users\j4\Projects\rxsupp.admin\test


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       03.12.2017     18:30            115 users-component_test.jsx
-a----       03.12.2017     18:27           9185 users_test.js

```

**Comment by eliperelman** [comment]
Sorry about your issue. I don't currently have a Windows machine, but I will see if I can get someone with Windows to replicate tomorrow. Thanks!

**Comment by sgillespie** [comment]
The issue on windows is that `path.join` uses a backslash as the separator, and the backslash escapes the following `.`. Therefore the regex won't match the path correctly. Here's the bit:

```
const testRegex = join(
    basename(tests),
    `.*(_test|_spec|\\.test|\\.spec)\\.(${extensions.join('|')})$`
);
```
One possible solution would be to join the two strings with `/` as the separator. I've tested this by overriding the `testRegex` and fixed the issue.

**Comment by constgen** [comment]
Guys, I have created this PR more then a week ago https://github.com/mozilla-neutrino/neutrino-dev/pull/679. Please merge

**Comment by constgen** [comment]
Is somebody going to merge fixes? My team have to modify `node_modules` manually to make it work. It is insane.

**Comment by sgillespie** [comment]
@constgen: You don't actually have to modify `node_modules`, just override `testRegex` in `neutrinorc.js`

```
    ['@neutrinojs/jest', {
      coverageReporters: ['lcov', 'text'],
      coveragePathIgnorePatterns: [
        '/node_modules/',
        '<rootDir>/src/index.js',
      ],
      setupTestFrameworkScriptFile: '<rootDir>/src/setup.test.js',
      testPathIgnorePatterns: ['setup.test.js', '/node_modules/'],
      testRegex: 'src/.*(_test|_spec|\\.test|\\.spec)\\.(js|jsx|vue|ts|tsx|mjs)$'
    }],
```

**Comment by constgen** [comment]
Thanks for advise. But I hope this nightmare will finish soon.

**Comment by sgillespie** [comment]
This has been released in v8.1.2. I don't have a windows setup to check it with, but as far as I can tell it should work fine (still works on linux :))

**Comment by eliperelman** [comment]
@sgillespie awesome, thank you!

**Comment by constgen** [comment]
Issue fixed. Checked on Windows.

**Comment by edmorley** [comment]
Fixed by #679, released in v8.1.2. Many thanks for everyone's help here :-)
